src/plugins/mvcc_delete_plugin.cpp

```diff
@@ -76,9 +82,16 @@ void MvccDeletePlugin::_logical_delete_loop() {
 
           std::unique_lock<std::mutex> lock(_mutex_physical_delete_queue);
           _physical_delete_queue.emplace(table, chunk_id);
+          removed_row_count += chunk_invalid_rows;
         }
       }
     }
+    if (removed_row_count > 0) {
+      std::ostringstream message;
+      message << "Compressed " << table_name << " by factor "
+              << std::to_string(static_cast<double>(removed_row_count) / row_count);
```

Comment:
"Compressed" is not the perfect term here. Maybe "Consolidated x chunks, freeing up y MB"?

Reply:
Hm, do we have a way to get a chunk's or segment's size? I guess this is not trivial when we consider string columns and different segment types, e.g. additional dictionaries (which can be different when we change the table order as the plugin does).

Reply:
estimate_memory_usage